stablehlo/reference/Ops.cpp

```diff
@@ -699,14 +699,13 @@ SmallVector<InterpreterValue> eval(Region &region,
 
       Sizes windowDimensions(rank, 1);
       if (auto windowDimensionsAttr = selectAndScatterOp.getWindowDimensions())
-        windowDimensions.assign(
-            windowDimensionsAttr->getValues<int64_t>().begin(),
-            windowDimensionsAttr->getValues<int64_t>().end());
+        windowDimensions.assign(windowDimensionsAttr->begin(),
+                                windowDimensionsAttr->end());
 
       Sizes windowStrides(rank, 1);
       if (auto windowStridesAttr = selectAndScatterOp.getWindowStrides())
-        windowStrides.assign(windowStridesAttr->getValues<int64_t>().begin(),
-                             windowStridesAttr->getValues<int64_t>().end());
+        windowStrides.assign(windowStridesAttr->begin(),
```

Comment:
Is this safe? Does this create multiple vectors?

Reply:
Should be just one vector. `getWindowStrides` will return a vector and `begin`/`end` will return iterators on the vector.

Or maybe I'm misunderstanding what you are alluding to?